src/main/kotlin/lotto/controller/LottoController.kt

```diff
@@ -29,26 +29,45 @@ class LottoController(
 
     private fun inputPayment(): Int {
         OutputView.printEnterPayment()
-        return InputView.inputPayment()
+        return InputView.inputNumber()
     }
 
     private fun purchaseLottos(inputPayment: Int): Lottos {
+        val manualLottoCount = inputManualLottoCount()
         val lottoCount = lottoGenerator.getLottoCount(inputPayment)
-        OutputView.printPurchaseCount(lottoCount)
+        require(lottoCount >= manualLottoCount) { "수동 구매 로또 수가 구입 금액을 초과하였습니다." }
 
-        val lottos = lottoGenerator.generateLottos(lottoCount)
+        val manualLottos = generateManualLottos(manualLottoCount)
+
+        OutputView.printPurchaseCount(manualLottoCount, lottoCount)
+        val lottos = lottoGenerator.generateLottos(lottoCount, manualLottos)
         OutputView.printLottos(lottos)
         return lottos
     }
 
+    private fun inputManualLottoCount(): Int {
+        OutputView.printEnterManualLottoCount()
+        return InputView.inputNumber()
+    }
+
+    private fun generateManualLottos(manualLottoCount: Int): MutableList<LottoNumbers> {
```

Comment:
```suggestion
    private fun generateManualLottos(manualLottoCount: Int): List<LottoNumbers> {
```

코틀린은 불변을 지키기 매우 좋은 언어이죠! List로 변경하여 불변을 지켜주는게 좋을 것 같습니다. 불변이라는게 더 이득이 되는 경우가 더 많기 때문이죠